angular2-jwt.ts

```diff
@@ -15,46 +15,54 @@ export interface IAuthConfig {
   noTokenScheme?:boolean;
 }
 
+export interface IAuthConfigOptional {
+    headerName?: string;
+    headerPrefix?: string;
+    tokenName?: string;
+    tokenGetter?: any;
+    noJwtError?: boolean;
+    globalHeaders?: Array<Object>;
+    noTokenScheme?: boolean;
+}
+
+export class AuthConfigConsts {
+    public static DEFAULT_TOKEN_NAME = 'id_token';
+    public static DEFAULT_HEADER_NAME = 'Authorization';
+    public static HEADER_PREFIX_BEARER = 'Bearer ';
+}
+
+const AuthConfigDefaults: IAuthConfig = {
+    headerName: AuthConfigConsts.DEFAULT_HEADER_NAME,
+    headerPrefix: null,
+    tokenName: AuthConfigConsts.DEFAULT_TOKEN_NAME,
+    tokenGetter: () => localStorage.getItem(AuthConfigDefaults.tokenName),
```

Comment:
Sorry, I don't know how to run the tests. They would have easily caught that.

tokenGetter now uses tokenName of the AuthConfigDefaults object.